.gitignore

```diff
@@ -1,31 +1,11 @@
-# Compiled class file
-*.class
-
-# Log file
-*.log
-
-# BlueJ files
-*.ctxt
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
```

Comment:
All these files should be stored in `target/` which is already excluded in .gitignore. Additionally we don't use BlueJ or J2ME. If it turns out we need that, we can add them again.
I would prefer to keep the file short and only add exclusions we really need to avoid surprises when files are ignored by accident.